### ISSUE TITLE
Consistent tab traversal

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#556]
Our GUI should be fully navigable with the tab key. Right now, that is distinctly *not* the case. wx.TAB_TRAVERSAL is only randomly used in one place, and implicitly via the default style for wx.Panel.

Ref #190 as well.
